apps/spruce/src/constants/welcomeModalProps.ts

```diff
@@ -1,72 +0,0 @@
-const newMainlineCommitsUser = [
-  {
-    img: "mainline_commits/carousel_1_05_06.png",
```

Comment:
Can you also please delete the associated media https://github.com/evergreen-ci/ui/blob/3c5d36a8708b71653bfd48cd23ac6c987f7936c1/apps/spruce/public/static/img/mainline_commits